sources/spanner/toddl.go

```diff
@@ -40,23 +40,28 @@ func (tdi ToDdlImpl) ToSpannerType(conv *internal.Conv, spType string, srcType s
 // conversion issues encountered.
 func toSpannerTypeInternal(conv *internal.Conv, srcType schema.Type) (ddl.Type, []internal.SchemaIssue) {
 	switch srcType.Name {
-	case "BOOL":
+	case "BOOL", "boolean":
 		return ddl.Type{Name: ddl.Bool}, nil
 	case "BYTES":
 		return ddl.Type{Name: ddl.Bytes, Len: srcType.Mods[0]}, nil
-	case "DATE":
+	case "bytea":
+		return ddl.Type{Name: ddl.Bytes, Len: ddl.MaxLength}, nil
+	case "DATE", "date":
 		return ddl.Type{Name: ddl.Date}, nil
-	case "FLOAT64":
+	case "FLOAT64", "double precision":
 		return ddl.Type{Name: ddl.Float64}, nil
-	case "INT64":
+	case "INT64", "bigint":
 		return ddl.Type{Name: ddl.Int64}, nil
-	case "JSON", "JSONB":
+	case "JSON", "jsonb":
 		return ddl.Type{Name: ddl.JSON}, nil
-	case "NUMERIC":
+	case "NUMERIC", "numeric":
 		return ddl.Type{Name: ddl.Numeric}, nil
-	case "STRING":
+	case "STRING", "character varying":
+		if len(srcType.Mods) == 0 {
+			return ddl.Type{Name: ddl.String, Len: ddl.MaxLength}, nil
+		}
```

Comment:
Is this relevant for `STRING` datatype of GSQL as well? If not, I would suggest moving `character varying` to its own case.